scripts/index.js

```diff
@@ -16,6 +16,26 @@ console.log(isEven(myNums));
 
 function squareNumbers(nums){
     let results = [];
-    nums.forEach();
+    nums.forEach(function(num){
+        results.push(num * num)
```

Comment:
Can you think of an alternative to pushing into a new array? (Hint: it's another array method, like `.filter`...)